helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/templates/reindexing-job.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.reindexing.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "alfresco-elasticsearch-connector.fullname" . }}-reindexing
+spec:
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "alfresco-elasticsearch-connector.selectorLabels" . | nindent 8 }}
+    spec:
+      # only set this secret if a private docker registry variable is defined
+      imagePullSecrets:
```

Comment:
I have to replace the string value with a variable. I'm going to replace it during the first commit after the pull request review.